src/stack.js

```diff
@@ -1,5 +1,43 @@
-'use strict'
-
 export default class Stack {
-  // your code here
+  constructor() {
+    this.size = 0
+    this.elements = []
+  }
+
+//stack.length()    // returns the number of elements in the stack.
+  length() {
+    return this.size
+  }
+
+//stack.isEmpty()   // returns true if the stack is empty or false if not.
+  isEmpty() {
+    return this.size === 0
+  }
+
+//stack.push("foo") // push an element (the string "foo") to the top of the stack.
+  push(elementToAdd) {
+    this.elementToAdd = elementToAdd
```

Comment:
I don't understand this implementation?  There's no need to store an instance variable with the last element added.  I'm not sure I see why you are calling `x.split()` if the element type is a `String`, not where `x` comes from.